### PR TITLE
Fix typo in docs

### PR DESCRIPTION
**Pull request by hsm207**
### Motivation and Context

  1. Why is this change required?
      Fix some typos in the python sample notebooks
  2. What problem does it solve?
      User experience
  3. What scenario does it contribute to?
      A new user reading the docs to learn how to use the project
  4. If it fixes an open issue, please link to the issue here.
       NA

### Description

Fix some typos and broken links.

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows SK Contribution Guidelines (https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md)
- [x] The code follows the .NET coding conventions (https://learn.microsoft.com/dotnet/csharp/fundamentals/coding-style/coding-conventions) verified with `dotnet format`
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

**Comment by hsm207** [comment]
@microsoft-github-policy-service agree company="Weaviate"

File: samples/notebooks/python/02-running-prompts-from-file.ipynb [hunk 1]
```diff
@@ -75,7 +75,7 @@
       "source": [
         "Given a semantic function defined by these files, this is how to load and use a file based semantic function.\n",
         "\n",
-        "Load and configure the kernel, as usual, loading also the AI service settings defined in the [Setup notebook](0-AI-settings.ipynb):"
+        "Load and configure the kernel, as usual, loading also the AI service settings defined in the [Setup notebook](01-basic-loading-the-kernel.ipynb):"
```
Comment (lemillermicrosoft) [inline samples/notebooks/python/02-running-prompts-from-file.ipynb:78]: I think `00-getting-started.ipynb` might be more appropriate, thoughts @alexchaomander ?

File: samples/notebooks/python/03-semantic-function-inline.ipynb [hunk 2]
```diff
@@ -38,11 +38,12 @@
       ]
     },
     {
+      "attachments": {},
       "cell_type": "markdown",
       "id": "d90b0c13",
       "metadata": {},
       "source": [
-        "Prepare a semantic kernel instance first, loading also the AI service settings defined in the [Setup notebook](0-AI-settings.ipynb):"
+        "Prepare a semantic kernel instance first, loading also the AI service settings defined in the [Setup notebook](01-basic-loading-the-kernel.ipynb):"
```
Comment (alexchaomander) [inline samples/notebooks/python/03-semantic-function-inline.ipynb:46]: same this should be `00-getting-started.ipynb`

File: samples/notebooks/python/02-running-prompts-from-file.ipynb (hunk 1 above)
Comment (alexchaomander) [reply samples/notebooks/python/02-running-prompts-from-file.ipynb:78]: Yeah for loading the kernel with AI service settings, `00-getting-started.ipynb` is a better place to start

**Review by alexchaomander — CHANGES_REQUESTED** [review]

File: samples/notebooks/python/02-running-prompts-from-file.ipynb (hunk 1 above)
Comment (hsm207) [reply samples/notebooks/python/02-running-prompts-from-file.ipynb:78]: done!

**Review by dluc — APPROVED** [review]

**Pull request merged**